docs/how-tos/rotating-secrets.md

```diff
@@ -0,0 +1,102 @@
+---
+title:  Rotating Secrets
+---
+
+Last updated: `Oct 6th, 2023`
+
+It is unfortunate yet inevitable that secrets such as API keys get leaked upon occasion. In most cases, this is an urgent matter that must be addressed immediately, but it depends on what information was leaked, whether it's still valid, and what damage can be done with that information.
+
+In the steps below, we'll walk through an example for how to resolve a possible leak of a Stripe API key.
+
+## Find or be informed about the (potential) leak
+
+There are many ways this can happen:
+* An engineer stumbles upon it while working on a patch
+* The Mozilla information security team finds it as part of a routine scan
+* It is flagged as part of Mozilla's bug bounty program
+* GitHub notifies us with a [secret scanning alert](https://docs.github.com/en/code-security/secret-scanning)
+* And more...
+
+In the case of our example, we received a secret scanning alert from GitHub in the `fxa` repo.
+
+## Confirm the leak and its severity
+
+Sometimes, sensitive looking data isn't actually valid or compromising. Try to answer the following questions, answered below with our Stripe API key example:
+
+* **Is this a valid secret?**
+:::note
+If the secret is expired or otherwise invalid, the urgency goes away, and you can skip to the next step. If the key is valid but only in sandbox/test environments with fake data, the urgency also goes away, but you'll still want to answer the questions below.
+:::
+  * Yes. It is listed in the [stage (`SUB_PLAT_STAGE`) dashboard](https://dashboard.stripe.com/test/apikeys) as the unrestricted [test-mode](https://stripe.com/docs/test-mode) secret key, and it is active. This is a real leak.
+* **Is it in use?**
+  * In this case, the [Stripe API Keys page in the dashboard](https://dashboard.stripe.com/test/apikeys) has helpful information including when a given valid key was last used. Our example key was last used earlier today, so it's definitely still in use!
+* **What do we use it for?**
+  * Since this is an unrestricted Stripe API key (test mode), it is used to make read/write API requests to any Stripe API in test mode in our staging environment. Without [searching for this key value explicitly in various places](#places-to-check-where-a-secret-might-be-used), one can only guess at this point that it's likely used by our stage apps to make Stripe API calls, and maybe by our CI end-to-end tests to do the same.
```

Comment:
This might be personal but from a SR perspective, I didn't relate this link:

> [searching for this key value explicitly in various places](#places-to-check-where-a-secret-might-be-used)

to this anchor:

> Places to check where a secret might be used

Is there perhaps a better way to make this obvious? Completely nit so not holding back merging, but maybe leave a note to fix in future?